.github/workflows/ci.yml

```diff
@@ -2,27 +2,29 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
 
 jobs:
   unit_tests:
```

Comment:
@binodpant is this line the unit tests that aren't running? I'm wondering if it's possible that the changes here are what broke it.

Reply:
hmm yeah I am not making the connection here yet. this section looks like we are just adding the strategy and I can see all the matrix test nodes have passed, this change looks very similar to what was done in a repo I recently worked on https://github.com/openedx/frontend-app-enterprise-public-catalog/pull/198/files

Reply:
May be the required checks settings need to updated.

Reply:
Got it. I'm not sure how to update the required checks but I think @justinhynes will know. Is this good to merge then?

Reply:
Yeah it is. Someone with access to repository settings needs to remove the outdated check(`unit_tests`) and mark the `CI / tests (16) (pull_request)` check required now and then we will be good to merge the PR.